hs_model_program/templates/pages/modelprogramresource.html

```diff
@@ -13,59 +13,58 @@
                     {########################}
                     {#       CONTENT FILES     #}
                     {########################}
-
-                    {% if mpmetadata.get_engine_list.items|first|first or mpmetadata.get_software_list.items|first|first or mpmetadata.get_releasenotes_list.items|first|first or mpmetadata.get_documentation_list.items|first|first %}
+                    {% if mpmetadata.get_engine_list or mpmetadata.get_software_list or mpmetadata.get_releasenotes_list or mpmetadata.get_documentation_list %}
                     <legend>Content files</legend>
                     <table class="custom-table">
                         <tbody>
-                        {% if mpmetadata.get_engine_list.items|first|first %}
-                            {% for name, path in mpmetadata.get_engine_list.items %}
+                        {% if mpmetadata.get_engine_list %}
```

Comment:
On these ones, if I don't include the `if`, the row with the heading shows up even when there are no files: 
![image](https://cloud.githubusercontent.com/assets/6943441/25192503/7129b3c0-2501-11e7-8057-1fb610398137.png)

Actually, when I was testing it, what I thought was working `{% if mpmetadata.get_engine_list %}` did not work and it needed to be `{% if mpmetadata.get_engine_list|first %}` resulting in what we want: 
![image](https://cloud.githubusercontent.com/assets/6943441/25192592/c319b3b0-2501-11e7-8985-37720a0e0b9a.png)